babel/src/test/resources/sql/big-query.iq

```diff
@@ -371,6 +371,30 @@ SELECT EXTRACT(DAY FROM DATE '2013-12-25') AS the_day;
 
 !ok
 
+SELECT
+  EXTRACT(DAYOFWEEK FROM DATE '2008-12-25') as day_of_week,
```

Comment:
This PR includes some changes which are not release to CALCITE-5565

Reply:
Just rebased, was previously dependent on uncommitted changes. Sorry about that!